SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -259,6 +262,7 @@ extension ViewController: UITextFieldDelegate {
 extension ViewController: PushKitEventDelegate {
     func credentialsUpdated(credentials: PKPushCredentials) {
         guard
+            registrationRequired(),
             let accessToken = fetchAccessToken(),
             UserDefaults.standard.data(forKey: kCachedDeviceToken) != credentials.token
         else { return }
```

Comment:
@timrozum can you please review this for the swiftyness of this.

Reply:
The style is pretty close but the format below is recommended for multiline guard. So I would just move the return to a new line in this case. FYI for single line guard it is common in Swift to put return on first line so that it is all just a one liner.

```swift
guard 
    let number1 = number1,
    let number2 = number2,
    let number3 = number3 
else {
    fatalError("impossible")
}
```
https://github.com/raywenderlich/swift-style-guide#golden-path